Tests/FunctionalTests/helpermethod.ts

```diff
@@ -57,25 +57,34 @@ export async function SearchAttributeSecondRow(page:Page, attributeName: string)
 //</summary>
 //<param> filesize </param>
 
-  export function  GetFileSizeInBytes(fileSize: string)
-  {
-        let fileSizeChar=fileSize.split(' ');       
-        
-        switch (fileSizeChar[1]) {
-          case "KB":
-            fileSizeInBytes=parseInt(fileSizeChar[0])*1024;
-            break;
-          case "MB":
-            fileSizeInBytes=parseInt(fileSizeChar[0])*1024*1024;
-            break;
-          case "GB":
-              fileSizeInBytes=parseInt(fileSizeChar[0])*1024*1024*1024
-            break;
-          }
-        
-      return fileSizeInBytes; 
```

Comment:
We've removed the `fileSizeInBytes` variable, please can we remove it's declaration from the top of the file

Reply:
Done. Thanks